common/src/main/java/com/wynntils/features/user/overlays/QuestInfoOverlayFeature.java

```diff
@@ -24,7 +24,7 @@
 import com.wynntils.wc.utils.scoreboard.quests.QuestManager;
 import java.util.List;
 
-@FeatureInfo(category = "overlays")
+@FeatureInfo(category = "Overlays")
```

Comment:
Btw, did we ever discuss if the categories should be i18n'ed, or made into like an enum so we keep a proper collection of what categories there are?

Reply:
I18nd enum was settled on iirc, but we never did it.